cmd/mayactl/app/command/volume.go

```diff
@@ -95,6 +95,14 @@ Examples:
 
  # Delete a Volume created in 'test' namespace:
    $ mayactl volume delete --volname <vol> --namespace test
+
+ # Resize a Volume created in default namespace:
+   $ mayactl volume resize --volname <vol> --size <size><unit>
```

Comment:
@mittachaitu let's add one resize test using `mayactl volume resize` for any Cstor volume we have in travis tests

Reply:
@prateekpandey14 istgt doesn't support resize, to support resize operation at istgt(https://github.com/mittachaitu/istgt/commit/25c02c152159336577d786460506916b59fc53c6) working on the UT to raise a PR.